modules/util/slice.go

```diff
@@ -88,3 +88,13 @@ func SliceRemoveAllFunc[T comparable](slice []T, targetFunc func(T) bool) []T {
 	}
 	return slice[:idx]
 }
+
+// SliceFind returns the element id of the first element that matches the condition, otherwise -1
+func SliceFind[T any](slice []T, condFunc func(T) bool) int {
```

Comment:
Maybe we can use `SliceContainsFunc` directly?

Reply:
I don't think so, it only returns bool, while we need the specific element or its id.